libs/langchain-google-common/src/types.ts

```diff
@@ -48,6 +48,7 @@ export interface GoogleAISafetySetting {
 export interface GoogleAIModelParams {
   /** Model to use */
   model?: string;
+  modelName?: string;
```

Comment:
Is "modelName" the preferred parameter name and "model" not?
Is there any reason to keep "model" around?

Reply:
No, wasn't sure if it was there for another reason. Removing now.